build/webpack.client.config.js

```diff
@@ -38,9 +39,6 @@ module.exports = function getClientConfig(configOpts) {
             optimization: {
                 minimizer: [
                     new TerserPlugin({
-                        cache: true,
-                        parallel: false,
-                        sourceMap: true,
```

Comment:
`cache:true` and `sourceMap:true` should happen by default now.

`parallel` will default to `true` now - this seems like an optimization we want to enable, so allowing that unless we see a reason not to.  Unsure what the reason we turned it off initially was

https://github.com/webpack-contrib/terser-webpack-plugin/blob/master/CHANGELOG.md#200-2019-09-05